python/README.md

```diff
@@ -0,0 +1,56 @@
+# This folder contains manual bindings for aikidopy using pybind11.
+
+0. The installation guide have been tested on
+
+- sudo apt installed python 2.7 and python 3.4.
+- anaconda managed python 3.6
+
+If you use system installed python, ensure you have `python2.7-dev` or `python3-dev` correspondingly, or run `sudo apt-get install python3-dev` to obtain it.
```

Comment:
I think this part about installing the dev headers is the actual step 0. The stuff before this about versions that have been tested is useful information, but aren't actually part of the checklist.